packages/e2e-tests/src/specs/editor/publishPanel/adminUser.js

```diff
@@ -82,7 +82,16 @@ describe('Publish panel in document tab', () => {
     await page.keyboard.type('auth');
 
     // wait for search results
-    await page.waitForTimeout(200);
+    await page.waitForResponse(
+      (response) =>
+        //eslint-disable-next-line jest/no-conditional-in-test
+        response.url().includes('web-stories/v1/users') &&
+        response.url().includes('search=auth') &&
+        response.status() === 200
+    );
+
+    // add small delay after we have results
+    await page.waitForTimeout(400);
```

Comment:
Re-added a delay here to test if that clears this test issue (again we need a proper solve) - either way a test is still failing with this in.